frontend/src/components/CountdownPage.js

```diff
@@ -50,6 +50,15 @@ export default function CountdownPage() {
   const difficulty = searchparams.get("difficulty");
   console.log("Received as: " + difficulty)
 
+  // Get Question ID
+  let parsedDifficulty = difficulty.toLowerCase();
+  async function getQuestionID(difficulty) {
+    const response = await fetch('http://localhost:3001/Questions/Difficulty/' + difficulty)
```

Comment:
can we abstract the local host fetch req and put the link in the constants.js file instead